Testsuite: gov.nasa.jpf.test.vm.basic.InitializeInterfaceClassObjectRefTest
Tests run: 1, Failures: 0, Errors: 0, Time elapsed: 0.528 sec
------------- Standard Output ---------------
  running jpf with args: +log.finest+=,gov.nasa.jpf.vm.ClassInfo
JavaPathfinder core system v8.0 (rev 28+) - (C) 2005-2014 United States Government. All rights reserved.


====================================================== system under test
gov.nasa.jpf.test.vm.basic.InitializeInterfaceClassObjectRefTest.runTestMethod()

====================================================== search started: 10/12/15 3:42 AM

====================================================== error 1
gov.nasa.jpf.vm.NoUncaughtExceptionsProperty
java.lang.RuntimeException: This test throws an expected exception.
	at gov.nasa.jpf.test.vm.basic.InitializeInterfaceClassObjectRefTest.test(InitializeInterfaceClassObjectRefTest.java:41)
	at java.lang.reflect.Method.invoke(gov.nasa.jpf.vm.JPF_java_lang_reflect_Method)
	at gov.nasa.jpf.util.test.TestJPF.runTestMethod(TestJPF.java:648)


====================================================== snapshot #1
thread java.lang.Thread:{id:0,name:main,status:RUNNING,priority:5,isDaemon:false,lockCount:0,suspendCount:0}
  call stack:
	at gov.nasa.jpf.util.test.TestJPF.runTestMethod(TestJPF.java:650)


====================================================== results
error #1: gov.nasa.jpf.vm.NoUncaughtExceptionsProperty "java.lang.RuntimeException: This test throws an ex..."

====================================================== search finished: 10/12/15 3:42 AM
------------- ---------------- ---------------

Testcase: test took 0.514 sec
